tests/tracking/fluent/test_metric_value_conversion_utils.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+
+from mlflow.exceptions import MlflowException
+from mlflow.tracking.metic_value_conversion_utils import *
+from tests.helper_functions import random_str, random_int
+
+import numpy as np
+import tensorflow as tf
+import torch
+
+import pyspark.ml.param
+import pyspark.ml.util
+
+
+class TestMetricValueConversionUtils(unittest.TestCase):
```

Comment:
Nit: Can we remove `unittest.TestCase` in favor of pytest-style test cases? For example, see https://github.com/mlflow/mlflow/blob/master/tests/models/test_model_input_examples.py.